include/kokkos-utils/atomics/InsertOp.hpp

```diff
@@ -11,7 +11,7 @@ namespace Kokkos::utils::atomics
 /**
  * @brief Insert an element in a view at a specific index using @c Kokkos::atomic_min.
  *
- * To be used with *e.g.* @ref Kokkos::UnorderedMap::insert.
+ * To be used with *e.g.* @c Kokkos::UnorderedMap::insert.
```

Comment:
Just wondering why you are avoiding all the `@ref`?

Reply:
It's failing even in a pipeline that doesn't include this PR:

- https://github.com/uliegecsm/kokkos-utils/actions/runs/9968853773